flame_utils/misc/confupdate.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import logging
+from flame import Machine
+
+import numpy as np
+
+_LOGGER = logging.getLogger(__name__)
+
+
+def conf_update(machine):
```

Comment:
Why do you need this function?

Reply:
*m.conf()* returns initial settings of the lattice.
If user change the lattice setting by using *reconfigure(i, setting)*,
*m.conf(i)* returns new setting, but *m.conf()* returns initial setting.
So, *conf_update()* apply current settings to the initial settings.

Reply:
I've been aware of that this function has some common part with `generate_latfile`, but not the same for handling duplicated elements?

Reply:
You're right.
I think, I can polish the part of `generate_latfile` by using this function.

Reply:
Additonal note: update `ModelFlame` class with `insert_element` method and correct `clone_machine` method.